npm/react-mount-utils/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Should we mark this package as private and not publish it?

Reply:
Yes, good idea.

Thanks for the early review, I'll get this one green today.

Reply:
Done.